rir/src/interpreter/interp.c

```diff
@@ -1544,13 +1591,7 @@ SEXP evalRirCode(Code* c, Context* ctx, SEXP env, unsigned numArgs) {
             SEXP body = ostack_at(ctx, 1);
             SEXP formals = ostack_at(ctx, 2);
             res = allocSExp(CLOSXP);
-
-            assert(isValidFunctionSEXP(body));
```

Comment:
This code updates the closure body to use the most optimized version of the function. However, this no longer makes any sense, since the body of a closure is a vtable.